corehq/apps/data_dictionary/management/commands/populate_case_prop_groups.py

```diff
@@ -31,3 +33,17 @@ def populate_case_prop_groups(domain):
         )
         case_prop.group_obj = group
         case_prop.save()
+
+
+def remove_out_of_sync_prop_and_groups():
+    # Remove properties have different group and group_obj fields
```

Comment:
nit: properties "that" have

Reply:
Fixed. [e48381a](https://github.com/dimagi/commcare-hq/pull/32799/commits/e48381ad475009f6d6e85d8d8aafdcb67b530ad7)